pasion_preciouskaye/index.html

```diff
@@ -0,0 +1,175 @@
+      <li>Get my Dream Job</li>
+      <li>Traveling with My Love ones</li>
+      <li>Buy our Own House</li>
+      <li>Make My Mother</li>
+    </ul>
+  </div>
+
+  <div class="your-comment">
+    <h2> Your Comment About My Goals</h2>
+     <ul id="comments_list">
+        <li> </li>
+      </ul>
+      <button id="sort_asc">Sort by Date Ascending</button>
+      <button id="sort_desc">Sort by Date Descending</button>
+  </div>
+   
+    <div class="images">
+      <h2>Gallery</h2>
+      <div class="image-item">
+      <img
+        src="https://w0.peakpx.com/wallpaper/430/970/HD-wallpaper-doraemon-art-cute.jpg" />
+      <img
+        src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQyAv_4V8XzSLF2lTUeMpXS7p0nH-8QomfB5zzxh9INxw&s"/>
+      <img
+        src="https://i.pinimg.com/originals/05/f6/05/05f605183b35d08386e122bf5a767d63.jpg" />
+      <img
+        src="https://yt3.googleusercontent.com/HskYwAGlO-VcPpnUNJcfTKBbFrj2LNrRbrm1Ht5cuHhc621f2Tgzf-deKf3uL0qMY8_KRApMoZg=s900-c-k-c0x00ffffff-no-rj"/>
+      <img
+        src="https://i.pinimg.com/236x/b7/91/a9/b791a92b8fc7819738019516d168ed25.jpg"/>
+      <img
+      src="https://i.pinimg.com/736x/69/5f/2f/695f2f95029f73e501cf28263b5aa492.jpg"/>
+     </div>
+    </div>
+
+    <div class="thoughts">
+      <h2>Face-toFace or Online Class</h2>
+        <p> better to be face to face class than online, because there 
+          are many advantage you'll be able to communicate with your 
+          instructors and classmates, and you'll be able to communicate with 
+          your classmates you will learn more and more because you can
+          see how your instructor is doing and you can hear your own instructor
+          speak forward. While online classes have many disadvantages like the 
+          students are just doing unrelated from class, but 
+          there are also some advantages. The disavantage students are often
+          lazy and when the camera is off they don't listen, sleep, or do 
+          anything else it is not a matter of the subject, because the 
+          interaction between the students and the instructor is less 
+          understandable and audible.
+        </p>
+    </div>
+
+    <div class="my-comment">
+      <form class="mycomment" id="comment" oninput="myComment()">
+        <h2>Comment</h2>
+          <label for="name">Name: </label>
+          <input type="text" id="name">
+          <p>Comment: </p>
+          <textarea name="ycomment" id="ycomment"
+            cols="35" rows="10"></textarea>
+          <button type="submit" id="button_comment" disabled>Submit</button>
+    </div>
+      </form>
+      
+      <script src="./scripts.js">
+
+      </script>
+    </div>
+
+    <div class="container">
+      <h2>Contact Me</h2>
+      <form>
+        <div class="email">
+          <label type="email">Email</label>
+          <input type="email" id="email" name="email" />
+        </div>
+
+        <div class="fullname">
```

Comment:
I forgot to mention this. Class names should be written in kebab case. All lower case separated by hyphen. For reference, please click on the link provided below to access further information.


```suggestion
        <div class="full-name">
```


[https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx- TPQxdLySow/edit#slide=id.g24da8de92d7_0_5]